requirements/docs-requirements.txt

```diff
@@ -1,67 +1,2 @@
-#
```

Comment:
Hey @kaapstorm 
Can you share what chnage happened in the requirements files?

I see in description
> Unlocks versions (not necessary when all you need is Sphinx).